packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -965,6 +965,10 @@
   "Markdown_Headers": "Allow Markdown headers in messages",
   "Markdown_SupportSchemesForLink": "Markdown Support Schemes for Link",
   "Markdown_SupportSchemesForLink_Description": "Comma-separated list of allowed schemes",
+  "Markdown_ColorSupport_Description": "Markdown Support for Colors",
+  "Markdown_No_Colors": "No colors",
+  "Markdown_IRC_Colors": "IRC Colors Only",
+  "Markdown_Full_Colors": "All colors",
```

Comment:
Also, add `Markdown_Colors` to the i18n and it can be as simple as `Markdown Colors` as the value.